ANDES-TechReport-EN/02_methods.Rmd

```diff
@@ -1,18 +1,18 @@
 
-A large component of the ecosystem surveys conducted by the Gulf, Québec, Maritimes, and Newfoundland and Labrador Regions includes the collection of physical, chemical, and biological oceanographic data in support of the \gls{azmp} [@Therriault-1998]. A CTD/Rosette system is deployed at a subset of fishing stations where vertical profiles of temperature, conductivity (salinity), dissolved oxygen, chlorophyll and other parameters are collected. Water samples (e.g., nutrients, dissolved oxygen, salinity) are collected at predetermined depths using the Rosette system and attached Niskin bottles, and plankton samples are also collected via vertical ring net tows. These data support annual "state of the ocean" reporting conducted by the [AZMP](https://www.dfo-mpo.gc.ca/science/data-donnees/azmp-pmza/index-eng.html), and are also used in various stock assessments and ecosystem research projects to provide additional ancillary variables for interpreting the captures data.
+A large component of the ecosystem surveys conducted by the Gulf, Québec, Maritimes, and Newfoundland and Labrador Regions includes the collection of physical, chemical, and biological oceanographic data in support of the \gls{azmp} [@Therriault-1998]. A CTD/Rosette system is deployed at a subset of fishing stations where vertical profiles of conductivity (salinity), temperature, dissolved oxygen, chlorophyll and other parameters are collected. Water samples (e.g., nutrients, dissolved oxygen, salinity) are collected at predetermined depths using the Rosette system and attached Niskin bottles, and plankton samples are also collected via vertical ring net tows. These data support annual "state of the ocean" reporting conducted by the [AZMP](https://www.dfo-mpo.gc.ca/science/data-donnees/azmp-pmza/index-eng.html), and are also used in various stock assessments and ecosystem research projects to provide additional ancillary variables for interpreting the captures data.
```

Comment:
Moved "Temperature" to preserve the order of the CTD acronyme (Conductivity, Temp, Dissolved O2)

Reply:
the "D" actually stands for depth